tests/tools/calendar_test.py

```diff
@@ -115,3 +117,61 @@ async def test_calendar_tool_crashes_if_too_many_days(httpd: HTTPServer):
     assert False
   except ValueError as e:
     assert str(e) == "next_days must be at most 366"
+
+class MockUpdatingCalendarHTTPRequestHandler(SimpleHTTPRequestHandler):  
+
+    """Custom handler to serve different calendar files for testing."""
+    
+    def do_GET(self):
+      if self.request_number == 0:  
+          ics_path = path.join(path.dirname(__file__), "..", "fixtures", "test-calendar-1.ics")  
+      else:  
+          ics_path = path.join(path.dirname(__file__), "..", "fixtures", "test-calendar-2.ics")  
```

Comment:
Did you forget to commit the `test-calendar-1.ics` and `test-calendar-2.ics` files?